curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc1798ff86c76b9248c6eb3.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-2
 
 The `h1` to `h6` heading elements are used to signify the importance of content below them. The lower the number, the higher the importance, so `h2` elements have less importance than `h1` elements. Only use one `h1` element per page and place lower importance headings below higher importance headings.
 
-Add an `h2` element below the `h1` element that says `Cat Photos`.
+Below the `h1` element, wrap `Cat Photos` text inside an `h2` element.
```

Comment:
I worry about the wrap part of the sentence. Maybe something like.

> Below the `h1` element, add an `h2` element and give it `Cat Photos` as its text.

The sentence is still a bit awkward I'll admit.